src/components/userGroups/UserGroup.tsx

```diff
@@ -848,11 +861,37 @@ export const DirectPermission: React.FC<DirectPermissionRow> = ({
                     : permission.accessType === ACCESS_TYPE_MAP.HELM_APPS
                     ? customRoles.possibleRolesMetaForHelm[value].value
                     : customRoles.possibleRolesMeta[value].value}
+                {ApproverPermission && permission.approver && ', Approver'}
                 {React.cloneElement(children[1])}
             </components.ValueContainer>
         )
     }
 
+    const handleApproverChange = () => {
+        handleDirectPermissionChange(APPROVER_ACTION, { name: APPROVER_ACTION.label })
+    }
+
+    const RoleMenuList = (props) => {
```

Comment:
Same here

Reply:
This is already moved to ApprovalNodeEdge, right?